src/navigations/root-stack.js

```diff
@@ -110,25 +112,23 @@ export const RootNavigator = ({currentScreen}) => {
           paddingTop: getPaddingTop(currentScreen)
         }}>
         <NetworkStatusIndicator hide={true} />
-        {/* <StatusBar translucent backgroundColor="white" /> */}
-        <KeyboardAvoidingView style={{flex: 1}} enabled={Platform.OS === 'ios'} behavior="padding">
-          <RootStack.Navigator
-            screenOptions={{
-              headerShown: false,
-              safeAreaInsets: {
-                top: 0,
-                bottom: 0,
-                left: 0,
-                right: 0
-              }
-            }}>
-            {initialStartup.id !== null && initialStartup.id !== '' ? (
-              <RootStack.Screen name="AuthenticatedStack" component={AuthenticatedNavigator} />
-            ) : (
-              <RootStack.Screen name="UnauthenticatedStack" component={UnauthenticatedNavigator} />
-            )}
-          </RootStack.Navigator>
-        </KeyboardAvoidingView>
+
+        <RootStack.Navigator
+          screenOptions={{
+            headerShown: false,
+            safeAreaInsets: {
+              top: 0,
+              bottom: 0,
+              left: 0,
+              right: 0
+            }
+          }}>
+          {initialStartup.id !== null && initialStartup.id !== '' ? (
+            <RootStack.Screen name="AuthenticatedStack" component={AuthenticatedNavigator} />
+          ) : (
+            <RootStack.Screen name="UnauthenticatedStack" component={UnauthenticatedNavigator} />
+          )}
+        </RootStack.Navigator>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `KeyboardAvoidingView` to a simple `View` component might cause issues with keyboard overlap on iOS devices. The `KeyboardAvoidingView` was likely used to prevent the keyboard from covering up content when it appears. If the new `KeyboardWrapper` component is handling this behavior, then this change should be fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->